vm-mock/vm.js

```diff
@@ -211,7 +213,21 @@ export default function createMockedABI(memory, createImports, instantiate, bina
 
       assembly_script_get_time(){
         return BigInt(Date.now());
-      }
+      },
+
+      assembly_script_mock_call(ptr) {
+        scCallMockStack.push(getArrayBuffer(ptr));
+      },
+
+      assembly_script_call(_address, method, _param, _coins) {
+
+        if(scCallMockStack.length) {
+          return newArrayBuffer(scCallMockStack.shift());
+        }
+        console.warn(`Warning: No mock defined for sc call to "${ptrToString(method)}". Returning empty data`);
```

Comment:
Why not panicking ?

Reply:
its another valid strategy! and i have nothing against it. so lets go for it !